feature-libs/organization/unit-order/core/model/index.ts

```diff
@@ -4,5 +4,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+export * from './augmented-core.model';
```

Comment:
move `export interface OrderHistoryQueryParams {` from `augmented-core.model.ts` to the empty file at the same level `unit-order.model.ts` 

and then export that instead of this augmented-core.model